feature-libs/cart/saved-cart/occ/adapters/occ-saved-cart.adapter.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ConverterService, OccEndpointsService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { SavedCartAdapter } from '../../core/connectors/saved-cart.adapter';
+
+@Injectable()
+export class OccSavedCartAdapter implements SavedCartAdapter {
+  constructor(
+    protected http: HttpClient,
+    protected occEndpoints: OccEndpointsService,
+    protected converter: ConverterService
+  ) {}
+
+  create(
```

Comment:
as mentioned previously, let's change this to saveCart as we're saving a cart and not creating one


example: https://github.com/SAP/spartacus/pull/11326/files#diff-8865624513d9824ad06a0e524683841250d9fe26537d90340c39c7ec6a8b3aaeR40